x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/jwt/JwtAuthenticatorIdTokenTypeTests.java

```diff
@@ -8,35 +8,24 @@
 package org.elasticsearch.xpack.security.authc.jwt;
 
 import org.elasticsearch.xpack.core.security.authc.jwt.JwtRealmSettings;
-import org.junit.Before;
 
 import java.text.ParseException;
 
 import static org.hamcrest.Matchers.containsString;
 
 public class JwtAuthenticatorIdTokenTypeTests extends JwtAuthenticatorTests {
 
-    private String fallbackSub;
-    private String fallbackAud;
-
-    @Before
-    public void beforeTest() {
-        doBeforeTest();
-        fallbackSub = null;
-        fallbackAud = null;
-    }
```

Comment:
This is a simplification of the test classes. The preparation is now moved to the superclass since it can already perform the conditional logic based on the `getTokenType()` call.